db-scheduler-ui-frontend/package.json

```diff
@@ -6,7 +6,6 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "postbuild": "mkdir -p dist/db-scheduler-ui && mv dist/* dist/db-scheduler-ui/ 2>/dev/null || true",
```

Comment:
Why is this removed?

Reply:
ah, related to making the copying of static tools work on any OS I assume

Reply:
Yes, this has not worked on windows...